ToyBox/classes/Infrastructure/CasterHelpers.cs

```diff
@@ -169,10 +170,34 @@ public static class CasterHelpers {
                 .Where(x => x.CharacterClass == spellbook.Blueprint.CharacterClass && (x.Archetype == null || unit.Progression.IsArchetype(x.Archetype))).Select(y => y.Spell)
                 .ToList();
 
-            return GetActualSpellsLearned(spellbook, level, spellsToIgnore);
+            Spellbook real = null;
+            if (unit.TryGetPartyMemberForLevelUpVersion(out var ch))
```

Comment:
Please adopt the brace on the same line as the if style used by the rest of the project